core/src/main/java/hudson/PluginManager.java

```diff
@@ -948,49 +948,51 @@ public void dynamicLoad(File arc, boolean removeExisting, @CheckForNull List<Plu
 
     @Restricted(NoExternalUse.class)
     public void start(List<PluginWrapper> plugins) throws Exception {
-        Jenkins.get().refreshExtensions();
```

Comment:
Have to **Hide whitespace changes** to see what is happening here. Risk of merge conflicts, `git blame` issues, etc. For cases like this I tend to recommend either leaving the original code at its original indentation (despite looking ugly), or introducing a helper method:

```java
public void start(List<PluginWrapper> plugins) throws Exception {
    try (ACLContext context = ACL.as(ACL.SYSTEM)) {
        doStart(plugins);
    }
}
private void doStart(List<PluginWrapper> plugins) throws Exception {
   // as before…
}
```

to minimize the diff.

Reply:
I see. I'll work with the indentation.